core-processor/src/ext.rs

```diff
@@ -106,6 +106,48 @@ pub struct ProcessorContext {
     pub rent_cost: u128,
 }
 
+impl Default for ProcessorContext {
```

Comment:
I believe we must move this impl under `test` cfg

Reply:
This `Default` may be useful not only for tests, and actually it's a little bit different with one used in test below

Reply:
> @breathx were there any specific intentions not to implement `Default`/`Debug`/`Clone` for `ProcessorContext`?

In general, it doesnt matter. But we should never use any of such implicit defaults for runtime, so I'd avoid it